ompi/mca/osc/ucx/osc_ucx_comm.c

```diff
@@ -380,6 +380,13 @@ int ompi_osc_ucx_put(const void *origin_addr, int origin_count, struct ompi_data
         }
     }
 
+    if (!((module->win_info_array[target]).rkey_init) && (target_count > 0)) {
```

Comment:
maybe add this as macro/common function?
also looks like two pairs of redundant () around !((module->win_info_array[target]).rkey_init)

Reply:
added macro